crates/loans/src/lib.rs

```diff
@@ -437,33 +438,14 @@ pub mod pallet {
     #[pallet::hooks]
     impl<T: Config> Hooks<T::BlockNumber> for Pallet<T> {
         fn on_runtime_upgrade() -> frame_support::weights::Weight {
-            let max_exchange_rate = crate::MaxExchangeRate::<T>::get();
-            if max_exchange_rate.is_zero() {
-                crate::MaxExchangeRate::<T>::put(Rate::from_inner(DEFAULT_MAX_EXCHANGE_RATE));
-            }
-            let min_exchange_rate = crate::MinExchangeRate::<T>::get();
-            if min_exchange_rate.is_zero() {
-                crate::MinExchangeRate::<T>::put(Rate::from_inner(DEFAULT_MIN_EXCHANGE_RATE));
-            }
-            T::DbWeight::get().reads_writes(2, 2)
+            migration::collateral_toggle::Migration::<T>::on_runtime_upgrade()
```

Comment:
Since you also added the migration to `Executive`, the migration will run twice. Also, it'll run for each runtime. I don't think that's a problem in your particular migration but with it already being in `Executive`, it shouldn't be required here

Reply:
Let's remove here, I'll double check with chopsticks that the migration runs